docs/stock/standalone-navigator.md

```diff
@@ -0,0 +1,20 @@
+Standalone Navigator
+====================
+
+The Standalone Navigator is a versatile Highcharts component, designed to synchronize and manipulate the viewing range across multiple charts. This tool enables users to easily control the range of one or more charts by interacting with a single navigator interface. It's particularly useful in dashboards and data analysis applications where viewing different data sets in parallel is essential.
+
+## Binding Charts to the Navigator
```

Comment:
I suggest adding one more paragraph here: how to create the navigator. And information which files should be loaded.